docs/content/configuration.md

```diff
@@ -631,13 +631,15 @@ services:
 bundles:
   authz:
     service: gcs
-    resource: 'bundle.tar.gz?alt=media'
+    resource: 'bundles%2fhttp%2fexample%2fbundle.tar.gz?alt=media'
```

Comment:
I don't think think the resource name should be changed just for the sake of using encoding. The previous example on line 500 is enough to do that, IMHO.

Reply:
Ok, I did that specifically because the Cloud Run example above actually uses the same resource as I mention here; there the encoding is not required. This, to me, dives home better the distinction between both types of GCP resources. But of course, we can change this back to the original.

Reply:
No, that's fine. Thanks for helping out with the docs!